src/app/dataset-view/additional-components/metadata-component/components/add-push-source/add-push-source.component.ts

```diff
@@ -76,11 +71,62 @@ export class AddPushSourceComponent extends BaseMainEventComponent {
         }),
     });
 
-    public get readForm(): FormGroup {
+    public initEditForm(): void {
+        this.trackSubscription(
+            this.editService
+                .getEventAsYaml(this.getDatasetInfoFromUrl(), this.queryParamName ?? "")
+                .subscribe((result) => {
+                    this.history = this.editService.history;
+                    if (this.unsupportedSourceName()) {
+                        this.zone.run(() => {
+                            this.navigationServices.navigateToPageNotFound();
+                        });
+                    }
+                    if (result) {
+                        this.eventYamlByHash = result;
+                        const currentPushSourceEvent = this.editService.parseEventFromYaml(this.eventYamlByHash);
+                        console.log("==>", currentPushSourceEvent);
```

Comment:
1) Kill it
2) Find a rule in ESLint documentation that detects console usages, and enable it in our configuration.

Reply:
added eslint rule